### PR TITLE
Gut repo and replace with node-npm-boilerplate

### PR DESCRIPTION
**Pull request by MikaAK**
**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other... Please describe: Everything change

**Does this PR introduce a breaking change?**
- [x] Yes
- [ ] No

If this PR contains a breaking change, please describe the following...

* Impact: Yarn can be used, everything is wiped
* Github Issue(s) this is regarding: #179 

**Other information**:
it was decided that an easier path would be to start fresh and build in things instead of going back to try and maintain.

### CODE REVIEW

File: tests/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import {assert} from 'chai'
+
+describe('Webpack Karma', function () {
+  it('is true', function () {
+    assert.isTrue(true)
```
Comment (carlos-) [inline tests/index.js]: really?

File: package.json [hunk 2]
```diff
@@ -1,64 +1,75 @@
...
+    "shx": "0.1.4",
+    "standard-changelog": "0.0.1",
+    "standard-version": "3.0.0"
+  },
+  "dependencies": {
+    "loader-utils": "^0.2.16",
+    "source-map": "^0.5.6",
+    "webpack-dev-middleware": "^1.8.4"
   },
-  "config": {
```
Comment (d3viant0ne) [inline package.json]: You still need this if we plan to use the changelog generation

File: package.json [hunk 3]
```diff
@@ -1,64 +1,75 @@
...
+    "serve": "node dist",
+    "setenv:dev": "npm config set production false",
+    "setenv:prod": "npm config set production true",
+    "start": "nodemon $2 --exec babel-node",
+    "test": "mocha -c -S -R spec 'tests' --compilers js:babel-register --check-leaks",
+    "test:export": "mocha -S -R mochawesome 'tests' --compilers js:babel-register --reporter-options reportDir=reports --check-leaks",
+    "test:watch": "mocha -w -c -S -R spec 'tests' --compilers js:babel-register --check-leaks"
   },
   "devDependencies": {
-    "add-stream": "^1.0.0",
```
Comment (d3viant0ne) [inline package.json]: Also need this if we plan on using the current changelog / release workflow

**Comment by d3viant0ne** [comment]
After a bit of discussion with @MikaAK we are going with the built in solution for changelog generation. I'll land it whenever the conflicts are resolved.

**Pull request merged**